pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/BaseMailboxReceiveOperator.java

```diff
@@ -131,6 +131,12 @@ protected void onEos() {
     }
   }
 
+  @Override
+  protected void sampleAndCheckInterruption() {
+    // mailbox receive operator uses passive deadline instead of the active one
```

Comment:
Might be useful to add _why_ it uses the passive instead of active deadline here?

Reply:
Added